### PR TITLE
WIP: Prague 2021 get ready blog post and schedule tweak

### PR DESCRIPTION
**Pull request by thatdocslady**
- DECISION: What do we do about closing ticket sales? I set it for Friday evening for now
- Do not merge until the Hopin links are sent from Ti.to

### CODE REVIEW

File: docs/conf/prague/2021/news/get-ready.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+=============================
+
+Hey everyone!
+
+We're just a few days away from the start of the conference, and we're getting excited to see you soon at our online venue for three days of content, collaboration, and conversation. We're happy to share with you some useful resources to help you prepare for the conference.
+
+Tickets sales closing on Friday, October 1
+------------------------------------------
+
+We wanted to let you know that we will be closing ticket sales on **Friday, Octover 1st at 20:00 CEST**. This will help the organizer team manage the platform registrations in time for the start of the conference, and help keep the cozy community feeling of the event.
```
Comment (plaindocs) [inline docs/conf/prague/2021/news/get-ready.rst]:
```suggestion
We wanted to let you know that we will be closing ticket sales on **Thursday, September 30th at 20:00 CEST**. This helps the organizer team manage the platform registrations in time for the start of the conference, and helps keep the cozy community feeling of the event. 
```

File: docs/conf/prague/2021/news/get-ready.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Hey everyone!
+
+We're just a few days away from the start of the conference, and we're getting excited to see you soon at our online venue for three days of content, collaboration, and conversation. We're happy to share with you some useful resources to help you prepare for the conference.
+
+Tickets sales closing on Friday, October 1
+------------------------------------------
+
+We wanted to let you know that we will be closing ticket sales on **Friday, Octover 1st at 20:00 CEST**. This will help the organizer team manage the platform registrations in time for the start of the conference, and help keep the cozy community feeling of the event. 
+
+If you were hoping to come to this years event, we recommend that you `get your tickets soon <https://www.writethedocs.org/conf/prague/2021/tickets/>`_.
```
Comment (plaindocs) [inline docs/conf/prague/2021/news/get-ready.rst]:
```suggestion
If you were hoping to come to this years event, we recommend that you `get your tickets before Thursday <https://www.writethedocs.org/conf/prague/2021/tickets/>`_. 
```

**Review by plaindocs — COMMENTED** [review]
One typo and some nits.

File: docs/conf/prague/2021/news/get-ready.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+   :tags: {{year}}, prague-{{year}}, tickets
+
+Get Ready for the Conference!
+=============================
+
+Hey everyone!
+
+We're just a few days away from the start of the conference, and we're getting excited to see you soon at our online venue for three days of content, collaboration, and conversation. We're happy to share with you some useful resources to help you prepare for the conference.
+
+Tickets sales closing on Friday, October 1
```
Comment (thatdocslady) [inline docs/conf/prague/2021/news/get-ready.rst]:
```suggestion
Tickets sales closing on Thursday, September 30th
```

File: docs/conf/prague/2021/news/get-ready.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+------------------------------------------
```
Comment (thatdocslady) [inline docs/conf/prague/2021/news/get-ready.rst]:
```suggestion
-------------------------------------------------
```

**Review by mxsasha — APPROVED** [review]
To be clear, Tito links not yet sent, on that now, after I update sponsor info in Hopin

**Review by plaindocs — APPROVED** [review]
LGTM

**Pull request merged**